pages/linux/pkginfo.md

```diff
@@ -1,6 +1,7 @@
 # pkginfo
 
 > Query the package database on a CRUX system.
+> More information: <https://crux.nu/Main/Handbook3-6>
```

Comment:
I am not sure if this is the correct link :)

Reply:
That link contains a mountain of information about CRUX, I probably should've linked https://crux.nu/Main/Handbook3-6#ntoc19, this one directly takes you to the part about pkginfo.

However, I noticed that this page is for CRUX 3.6, and it will get really tedious to have to approve pull requests for future versions of CRUX in case of any breakage. So, I also found https://manned.org/pkginfo.1, since I saw in one other pull request/issue tracker (I don't remember which one off the top of my head) that manned.org was the preferred resource for man page links. 

Which one should I put in?

Reply:
I've decided to put in the manned.org page since, strictly for more information, the man page would be better.